paddle/fluid/platform/profiler.cc

```diff
@@ -225,6 +198,15 @@ RecordBlock::~RecordBlock() {
   ClearCurBlock();
 }
 
+static void BuildVar(const std::string& param_name,
+                     std::initializer_list<const char*> arguments,
+                     paddle::framework::proto::OpDesc::Var* var) {
```

Comment:
It seems that `BuildVar` is not used in this PR.

Reply:
mistake, removed.